projects/storefrontlib/src/lib/store-finder/components/store-finder-store-description/store-finder-store-description.component.spec.ts

```diff
@@ -71,6 +72,7 @@ describe('StoreFinderStoreDescriptionComponent', () => {
         StoreModule.forRoot({}),
         StoreModule.forFeature('stores', fromReducers.reducers)
       ],
+      schemas: [NO_ERRORS_SCHEMA],
```

Comment:
We have a backlog ticket to get rid of no_errors_schema because they are hiding an actual flaw in the test configuration (I believe you have used this schema because you had some errors in configuration).